osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/EventNotificationMessageService.java

```diff
@@ -58,18 +58,19 @@
     public void handleEvent(final String deviceIdentification, final Date dateTime, final EventType eventType,
             final String description, final Integer index) throws UnknownEntityException {
 
-        // Lookup device
-        final Device device = this.eventNotificationHelperService.findDevice(deviceIdentification);
+        // // Lookup device
+        // final Device device =
+        // this.eventNotificationHelperService.findDevice(deviceIdentification);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)